[2024-09-15T20:51:29.838+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:51:28.949373+00:00 [queued]>
[2024-09-15T20:51:29.848+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:51:28.949373+00:00 [queued]>
[2024-09-15T20:51:29.848+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 11
[2024-09-15T20:51:29.860+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-15 20:51:28.949373+00:00
[2024-09-15T20:51:29.864+0000] {standard_task_runner.py:57} INFO - Started process 485 to run task
[2024-09-15T20:51:29.867+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'manual__2024-09-15T20:51:28.949373+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpq5qxnwc0']
[2024-09-15T20:51:29.868+0000] {standard_task_runner.py:85} INFO - Job 54: Subtask raw_ingestion
[2024-09-15T20:51:29.970+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion manual__2024-09-15T20:51:28.949373+00:00 [running]> on host airflow-worker
[2024-09-15T20:51:30.143+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-15T20:51:28.949373+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-15T20:51:28.949373+00:00'
[2024-09-15T20:51:30.144+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-15T20:51:30.144+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-15T20:51:30.170+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-15T20:51:30.170+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-15T20:51:30.179+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-15T20:51:30.269+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-15T20:51:30.270+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-15T20:51:32.376+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkContext: Running Spark version 3.3.1
[2024-09-15T20:51:32.390+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-09-15T20:51:32.445+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceUtils: ==============================================================
[2024-09-15T20:51:32.446+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-09-15T20:51:32.446+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceUtils: ==============================================================
[2024-09-15T20:51:32.446+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkContext: Submitted application: load
[2024-09-15T20:51:32.459+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-09-15T20:51:32.466+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceProfile: Limiting resource is cpu
[2024-09-15T20:51:32.467+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-09-15T20:51:32.499+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SecurityManager: Changing view acls to: root
[2024-09-15T20:51:32.500+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SecurityManager: Changing modify acls to: root
[2024-09-15T20:51:32.500+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SecurityManager: Changing view acls groups to: 
[2024-09-15T20:51:32.501+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SecurityManager: Changing modify acls groups to: 
[2024-09-15T20:51:32.501+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(root); groups with view permissions: Set(); users  with modify permissions: Set(root); groups with modify permissions: Set()
[2024-09-15T20:51:32.679+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO Utils: Successfully started service 'sparkDriver' on port 39391.
[2024-09-15T20:51:32.704+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkEnv: Registering MapOutputTracker
[2024-09-15T20:51:32.727+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkEnv: Registering BlockManagerMaster
[2024-09-15T20:51:32.739+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-09-15T20:51:32.739+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-09-15T20:51:32.742+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-09-15T20:51:32.758+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-2c4fcfc5-98ff-4708-b136-f64c44e2042a
[2024-09-15T20:51:32.770+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-09-15T20:51:32.781+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-09-15T20:51:32.894+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2024-09-15T20:51:32.900+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO Utils: Successfully started service 'SparkUI' on port 4041.
[2024-09-15T20:51:32.974+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO Executor: Starting executor ID driver on host spark-master
[2024-09-15T20:51:32.980+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-09-15T20:51:32.995+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 34237.
[2024-09-15T20:51:32.995+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO NettyBlockTransferService: Server created on spark-master:34237
[2024-09-15T20:51:32.996+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-09-15T20:51:33.000+0000] {ssh.py:524} INFO - 24/09/15 20:51:32 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, spark-master, 34237, None)
[2024-09-15T20:51:33.002+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO BlockManagerMasterEndpoint: Registering block manager spark-master:34237 with 434.4 MiB RAM, BlockManagerId(driver, spark-master, 34237, None)
[2024-09-15T20:51:33.003+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, spark-master, 34237, None)
[2024-09-15T20:51:33.004+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, spark-master, 34237, None)
[2024-09-15T20:51:33.517+0000] {ssh.py:524} INFO - Traceback (most recent call last):
[2024-09-15T20:51:33.517+0000] {ssh.py:524} INFO -   File "/opt/shared/scripts/raw_ingestion.py", line 51, in <module>
[2024-09-15T20:51:33.519+0000] {ssh.py:524} INFO -     print(corredores.count())
[2024-09-15T20:51:33.520+0000] {ssh.py:524} INFO - TypeError: list.count() takes exactly one argument (0 given)
[2024-09-15T20:51:33.544+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO SparkContext: Invoking stop() from shutdown hook
[2024-09-15T20:51:33.550+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO SparkUI: Stopped Spark web UI at http://spark-master:4041
[2024-09-15T20:51:33.560+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-09-15T20:51:33.568+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO MemoryStore: MemoryStore cleared
[2024-09-15T20:51:33.569+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO BlockManager: BlockManager stopped
[2024-09-15T20:51:33.572+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-09-15T20:51:33.574+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-09-15T20:51:33.580+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO SparkContext: Successfully stopped SparkContext
[2024-09-15T20:51:33.580+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO ShutdownHookManager: Shutdown hook called
[2024-09-15T20:51:33.581+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO ShutdownHookManager: Deleting directory /tmp/spark-520e67f9-f890-470f-b980-8eb2f6e6eeb0
[2024-09-15T20:51:33.583+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO ShutdownHookManager: Deleting directory /tmp/spark-5e2ff880-83be-4b38-a657-82ad3ea73f14
[2024-09-15T20:51:33.585+0000] {ssh.py:524} INFO - 24/09/15 20:51:33 INFO ShutdownHookManager: Deleting directory /tmp/spark-520e67f9-f890-470f-b980-8eb2f6e6eeb0/pyspark-68c3a95d-01ad-4b2a-9d63-d6434d77f8bd
[2024-09-15T20:51:33.672+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 166, in run_ssh_client_command
    self.raise_for_status(exit_status, agg_stderr, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 160, in raise_for_status
    raise AirflowException(f"SSH operator error: exit status = {exit_status}")
airflow.exceptions.AirflowException: SSH operator error: exit status = 1
[2024-09-15T20:51:33.674+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240915T205128, start_date=20240915T205129, end_date=20240915T205133
[2024-09-15T20:51:33.689+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 54 for task raw_ingestion (SSH operator error: exit status = 1; 485)
[2024-09-15T20:51:33.730+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-15T20:51:33.767+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
